scripts/process_data.py

```diff
@@ -264,6 +278,12 @@ def main(self) -> None:
             self.data, image_dir=image_dir, num_frames_target=self.num_frames_target, verbose=self.verbose
         )
 
+        # Generate planar projections if equirectangular
+        if self.camera_type == "equirectangular":
+            image_dir = process_data_utils.generate_planar_projections_from_equirectangular(
+                image_dir, (1200, 1200), self.images_per_equirect
```

Comment:
Is hardcoding 1200x1200 resolution the right way to go here? Seems that it should be dependent on the input resolution...

Reply:
Matt suggested I do that since it gets down sampled for training anyway. What would you recommend though?

Reply:
- Does power of two matter/help, if downsampling?
- Sounds like Matt's suggesting no higher than that, which makes sense currently; I'm wondering whether people will try with 360 videos with less than 4K resolution, for which 1200x1200 might actually be larger than necessary.

Reply:
Maybe it begs the question overall of: what is the highest image input resolution that nerfstudio will effectively use?  Not just an equirectangular-input question.

Reply:
do you think it should be set to like a proportion?

Reply:
I'm not sure of a great way to choose the resolution. I agree that 1200 might be too much in many cases. One possible heuristic would be
`num_image * res**2 = orig_height*orig_width`

Reply:
I just tested out this heuristic, it works very well (n=1)